packages/terra-clinical-modal-manager/src/ModalManager.jsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Modal from 'terra-modal';
+import 'terra-base/lib/baseStyles';
+
+import AppDelegate from 'terra-clinical-app-delegate';
+import SlideGroup from 'terra-clinical-slide-group';
+import getBreakpoints from 'terra-responsive-element/lib/breakpoints';
+
+import './ModalManager.scss';
+
+const propTypes = {
+  /**
+   * The AppDelegate instance provided by the containing component. If present, its properties will propagate to the children components.
+   **/
+  app: AppDelegate.propType,
+
+  /**
+   * Components that will receive the ModalManager's AppDelegate configuration. Components given as children must appropriately handle an `app` prop.
+   **/
+  children: PropTypes.node,
+
+  /**
+   * From `connect`. The Array of component keys that will be used to instantiate the Modal's inner components.
+   **/
+  modalComponentKeys: PropTypes.array,
```

Comment:
I feel like requiring the consumer to pass in both the keys and array of data with the matching key could make it more complicated/greater room for error within the `buildModalComponents`. Is there a reason not to use just the modalComponentData for building the components?